modules/store/src/action_creator.ts

```diff
@@ -25,6 +25,61 @@ export function createAction<
   type: T,
   creator: Creator<P, DisallowTypeProperty<R>>
 ): FunctionWithParametersType<P, R & TypedAction<T>> & TypedAction<T>;
+/**
+ * @description
+ * Creates a configured `Creator` function that, when called, returns an object in the shape of the `Action` interface.
+ *
+ * Action creators reduce the explicitness of class-based action creators.
+ *
+ * @param type Describes the action that will be dispatched in your application
+ * @param config Additional metadata needed for the handling of the action.  See {@link createAction#usage-notes Usage Notes}.
+ *
+ * @usageNotes
+ *
+ * **Declaring an action creator**
+ *
+ * Without additional metadata:
+ * ```ts
+ * export const increment = createAction('[Counter] Increment');
+ * ```
+ * With additional metadata:
```

Comment:
Can we also add the action creator with a function here?

```
export const loginSuccess = createAction(
  '[Auth/API] Login Success',
   (user: User) => user.id
);
```

Reply:
Added in dfbeafc.